test/upgrade-matrix/mzcompose.py

```diff
@@ -46,10 +46,10 @@
     Redpanda,
 )
 from materialize.mzcompose.services import Testdrive as TestdriveService
-from materialize.util import MzVersion, released_materialize_versions
+from materialize.util import MzVersion
+from materialize.version_list import VersionsFromDocs
 
-# All released Materialize versions, in order from most to least recent.
-all_versions = released_materialize_versions()
+all_versions = VersionsFromDocs().minor_versions()
```

Comment:
Optional: Consider renaming to `all_versions_in_ascending_order` or similar / shorter (also in other files where sensible)